jsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "paths": {
-      "@/*": ["./src/*"]
```

Comment:
Revert back. Don't make unnecessary changes in any config file without permission